examples/web/static/sandbox.js

```diff
@@ -0,0 +1,216 @@
+document.addEventListener("DOMContentLoaded", () =>
+{
+    const log = console.log;
+    let endpoint = undefined;
+
+    let popId = "";
+    let popKey = "";
+    let isStaging = false;
+    let isSandbox = false;
+
+
+    const inferStringInput = document.getElementById("infer-string");
+    const modelManifestInput = document.getElementById("model-manifest");
+    const modelListInput = document.getElementById("models-list");
+    const consoleOutputInput = document.getElementById("console-output");
+    const runButton = document.getElementById("submit-button");
+    const populateButton = document.getElementById("populate-button");
+    const popIdInput = document.getElementById("pop-id");
+    const popKeyInput = document.getElementById("pop-key");
+    const stagingCheckbox = document.getElementById("staging-checkbox");
+    const sandboxCheckbox = document.getElementById("sandbox-checkbox");
+
+
+    console.log("EyePop Sandbox Example");
+
+    console.log = (...args) =>
+    {
+        log.apply(console, args);
+        consoleOutputInput.value += args.join(" ") + "\n";
+    };
+    console.warn = (...args) =>
+    {
+        consoleOutputInput.value += "--Warning--: " + args.join(" ") + "\n";
+    };
+
+    consoleOutputInput.value = "Console Output: \n";
+
+    modelManifestInput.value = JSON.stringify(
+        [
+            { "authority": "legacy", "manifest": "https://s3.amazonaws.com/models.eyepop.ai/releases/legacy/1.2.0/manifest.json" },
```

Comment:
I like the example in general, but it exposes a lot of sausage-making and we don't intend to make this public. Please remove the example.